extension/fluentbitextension/process_linux.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build linux
+
+package fluentbitextension
+
+import (
+	"os/exec"
+	"syscall"
+)
+
+func applyOSSpecificCmdModifications(cmd *exec.Cmd) {
```

Comment:
Does this not work on darwin or any other os?

Reply:
Yep, just Linux for the initial version.  We can add support for Mac/Windows later if this proves to be a desirable approach to having the collector manage fluentbit.